hv_vline~-help.pd

```diff
@@ -0,0 +1,40 @@
+#N canvas 1159 431 540 482 10;
+#X text 47 26 [line~] based replacement for [vline~];
```

Comment:
Can you add a note saying that this object isn't doing any block boundary calculation or interpolation, so the output could be very slightly different